Dockerfile.ocp

```diff
@@ -0,0 +1,19 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
+
+COPY . /go/src/sigs.k8s.io/metrics-server
+
+WORKDIR /go/src/sigs.k8s.io/metrics-server
+RUN make metrics-server-ocp -f Makefile.ocp
+
+FROM registry.ci.openshift.org/ocp/4.14:base
+
+COPY --from=builder /go/src/sigs.k8s.io/metrics-server/metrics-server /usr/bin/metrics-server
+
+LABEL io.k8s.display-name="OpenShift Metrics Server" \
+      io.k8s.description="This is a component of OpenShift Container Platform that provides the Kubernetes resource metrics API based on Kubelet metrics" \
+      io.openshift.tags="openshift" \
+      summary="This is a component of OpenShift Container Platform that provides the Kubernetes resource metrics API based on Kubelet metrics" \
+      maintainer="OpenShift Monitoring Team <team-monitoring@redhat.com>"
+
+USER 1001
```

Comment:
Just curious about why does the user or UID changes from the other provided Dockerfile, is there any specific reason for this? IIUC the user 65534 which is mapped in the other Dockerfile is the 'nobody' one, used for unprivileged access.

Reply:
aah thats a good catch, I was referring to [prometheus-adapter ocp file](https://github.com/openshift/k8s-prometheus-adapter/blob/master/Dockerfile.ocp) for making this one so had used same UID. Looking at https://github.com/openshift/k8s-prometheus-adapter/blob/master/Dockerfile#L20 not sure if 1001 was used before updating in upstream. I will update to use `65534`